modules/backend/classes/FormField.php

```diff
@@ -461,6 +461,12 @@ protected function filterTriggerAttributes($attributes, $position = 'field')
         $triggerField = array_get($this->trigger, 'field');
         $triggerCondition = array_get($this->trigger, 'condition');
 
+        $triggerFieldGlobal = false;
```

Comment:
Why do you set this flag up here separate from where you actually use it instead of right by where it gets used?

Reply:
Mainly because I prefer setting variables at the beginning of the function but also because I want to get rid of the '@' in the ```$triggerField``` as soon as possible, since later there may be code added which may fail if '@' is present. Not sure if it makes much of a difference though in terms of readability: all ```$trigger*``` variables in this method are set at the beginning but used only later.

Reply:
Fair enough. Did you go through any other flags to indicate it's a global field other than `@`? As a side thought, how difficult do you think it would be to target fields that are a specified level above the current field? I.e. repeater in a repeater, wants to target a field on the parent repeater? Would that be feasible?

Reply:
Good idea. Just implemented and tests setting a number of '@' symbols to indicate how high in the hierarchy to go. For example: ```@type``` would indicate immediate parent, ```@@type``` would indicate parent's parent, etc. All works fine, so I could upgrade, however, do You have any better ideas for the syntax? I'm rather new to October, so not much on general conventions and syntax "habbits" here. If multiple '@' symbols are fine, then I'll amend

Reply:
I don't necessarily mind the `@` symbol, but I ask because it's used by component partials to indicate to the parser to use an overridable partial path: http://octobercms.com/docs/plugin/components#component-partials

Other symbols used include `~` and `$` in some file path parsing areas where they mean application path and plugin directory path respectively.

All that to say I'm not against it, just wondering if there's a better option out there. Perhaps a caret? `^`?

Reply:
I like the caret ```^```. It kinda points "up" and also has a similar meaning in git syntax which is normally quite familiar to developers. Ok, thanks for the tip, I will amend the PR